Improve wording (#96)
Ghostkeeper/SettingsGuide: resources/articles/platform_adhesion/raft_speed.md

Raft Print Speed
====

> This setting determines the overall speed at which the raft is printed. The actual speed at which the raft is printed may differ from this if specific speeds are set for the base, middle or top layers. 

Reviewer: Can we keep the word "still" in, to indicate the contrast like the original sentence did?

> This setting determines the overall speed at which the raft is printed. The actual speed at which the raft is printed may differ from this if specific speeds are set for the base, middle or top layers. 

Author: ```suggestion
This setting determines the overall speed at which the raft is printed. The actual speed at which the raft is printed may differ from this setting if specific speeds are set for the base, middle or top layers. 
```

Author: How about the above?
We could do **"may still differ from this setting"** if you think that is better.

> Printing the raft more slowly than the rest of the print is generally beneficial. Printing slower allows the plastic to anneal more because it stays hot for longer, which reduces the internal stresses within the lines and reduces warping. Because it stays hot for longer, the material can also flow out more, increasing the contact area with the bed which improves adhesion. In general, your raft will be stiffer, stronger, more reliable and stick better.

Author: ```suggestion
Printing the raft more slowly than the rest of the print is generally beneficial. Printing slower allows the plastic to anneal more because it stays hot for longer, which reduces the internal stresses within the lines and reduces warping. Because it stays hot for longer, the material can also flow out more, increasing the contact area with the bed which improves adhesion. In general, your raft will be stiffer, stronger, more reliable and stick better. However, the raft will take more time to print.
```

> However, of course, it will then take more time to print the raft.

Reviewer: I don't think this really improves wording, since it makes the sentence read very jaggedly.

How about this?

```suggestion
However, it will then naturally take more time to print the raft.
```

Author: I agree the sentence still reads poorly. Please see my new proposal  which I've added to the line above (_line 5_), since I don't think we  actually need a new paragraph.
